src/components/studentEngagementOpportunities.scss

```diff
@@ -0,0 +1,201 @@
+.student-engagement {
+  display: flex;
+  margin-bottom: var(--sectionSpacingSecondary);
+}
+
+.student-engagement__left,
+.student-engagement__right {
+  flex: 0 1 50%;
+  display: flex;
+  align-items: center;
+}
+
+.gallery-wrapper {
+  width: 100%;
+}
+
+.gallery {
+  width: 100%;
+  height: 100%;
+}
+
+.gallery-container {
+  width: 100%;
+  height: 100%;
+  position: relative;
+}
+
+.gallery-image-wrapper {
+  width: 100%;
+  padding-top: 66%;
+  position: relative;
+}
+
+.gallery-image {
+  width: 90%;
+  height: 90%;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  position: absolute;
+}
+
+.gallery-image__caption {
+  position: absolute;
+  bottom: 0;
+  height: auto;
+  background-color: #ffce34;
+  padding: var(--spacing-s);
+  width: 100%;
+}
+
+.gallery-image__caption--no-display {
+  display: none;
+}
+
+.gallery-image__title {
+  font-family: var(--ibmPlexMono);
+  font-size: 18px;
+  letter-spacing: 0.5px;
+  line-height: var(--mobileParagraphLineHeight);
+  margin-bottom: 8px;
+}
+
+.gallery-image__1 {
+  background-image: url('../images/1.jpg');
```

Comment:
Not sure if you updated this in another PR, but these images are provided in Contentful, make sure to pull them from there.